compiler/compiler/src/main/java/org/robovm/compiler/config/Config.java

```diff
@@ -234,7 +231,7 @@ protected Config() throws IOException {
                 new ObjCBlockPlugin(),
                 new AnnotationImplPlugin(),
                 new LambdaPlugin(),         
-                new ShadowFramePlugin()
+                new DebugInformationPlugin()
```

Comment:
debug information is injected during soot pass and lands into .ll files. 
then ClassCompiler picks this information up from llvm,check lines around this comments
 - Read out line number info from the .o file if any and assemble into a separate .o file.

and creates separate classname.linetable.o file for every class. also it set lineable reference in method structure where this information is picked up during runtime once stack trace is created. 
all but debug info injection was available from original RoboVM 1.8